From d3ebed319edbb450d31f3271d90216c332cf3177 Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <berndkuhls@hotmail.com>
Date: Thu, 13 Mar 2014 23:18:07 +0100
Subject: [PATCH 05/10] pkgconf: rewrite variable sdkdir, needed by
 xdriver_xf86-video-intel

- rename patch file to include the apply order

Signed-off-by: Bernd Kuhls <berndkuhls@hotmail.com>
---
 package/pkgconf/pkgconf-01-fix-variable.patch | 29 +++++++++++++++++++++++++++
 package/pkgconf/pkgconf-fix-variable.patch    | 27 -------------------------
 2 files changed, 29 insertions(+), 27 deletions(-)
 create mode 100644 package/pkgconf/pkgconf-01-fix-variable.patch
 delete mode 100644 package/pkgconf/pkgconf-fix-variable.patch

diff --git a/package/pkgconf/pkgconf-01-fix-variable.patch b/package/pkgconf/pkgconf-01-fix-variable.patch
new file mode 100644
index 0000000..4c213d6
--- /dev/null
+++ b/package/pkgconf/pkgconf-01-fix-variable.patch
@@ -0,0 +1,29 @@
+[PATCH] prefix sysroot to include/libdir path variables
+
+Prefix includedir / libdir variable values with sysroot if a variable is
+requested (--variable=<name>), similar to how it's done for -I / -L flags.
+
+This is sometimes used to find header files (E.G. in gst-plugins configure),
+so ensure the sysroot'ed files are used.
+
+Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
+Signed-off-by: Bernd Kuhls <berndkuhls@hotmail.com>
+
+diff -Nura pkgconf-0.8.9.orig/main.c pkgconf-0.8.9/main.c
+--- pkgconf-0.8.9.orig/main.c	2012-10-24 14:32:08.236508699 -0300
++++ pkgconf-0.8.9/main.c	2012-10-24 14:54:36.771070217 -0300
+@@ -298,7 +298,13 @@
+ 	if (eflag != PKG_ERRF_OK)
+ 		return false;
+ 
+-	printf("%s\n", req.buf);
++	if ( !strcmp(req.variable, "includedir") ||
++	  !strcmp(req.variable, "mapdir") ||
++	  !strcmp(req.variable, "sdkdir") ||
++	  !strcmp(req.variable, "libdir"))
++		printf("%s%s\n", sysroot_dir, req.buf);
++	else
++		printf("%s\n", req.buf);
+ 	return true;
+ }
+ 
diff --git a/package/pkgconf/pkgconf-fix-variable.patch b/package/pkgconf/pkgconf-fix-variable.patch
deleted file mode 100644
index e19f76c..0000000
--- a/package/pkgconf/pkgconf-fix-variable.patch
+++ /dev/null
@@ -1,27 +0,0 @@
-[PATCH] prefix sysroot to include/libdir path variables
-
-Prefix includedir / libdir variable values with sysroot if a variable is
-requested (--variable=<name>), similar to how it's done for -I / -L flags.
-
-This is sometimes used to find header files (E.G. in gst-plugins configure),
-so ensure the sysroot'ed files are used.
-
-Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
-
-diff -Nura pkgconf-0.8.9.orig/main.c pkgconf-0.8.9/main.c
---- pkgconf-0.8.9.orig/main.c	2012-10-24 14:32:08.236508699 -0300
-+++ pkgconf-0.8.9/main.c	2012-10-24 14:54:36.771070217 -0300
-@@ -298,7 +298,12 @@
- 	if (eflag != PKG_ERRF_OK)
- 		return false;
- 
--	printf("%s\n", req.buf);
-+	if ( !strcmp(req.variable, "includedir") ||
-+	  !strcmp(req.variable, "mapdir") ||
-+	  !strcmp(req.variable, "libdir"))
-+		printf("%s%s\n", sysroot_dir, req.buf);
-+	else
-+		printf("%s\n", req.buf);
- 	return true;
- }
- 
-- 
1.8.3.2

